static/app/components/onboarding/gettingStartedDoc/sdkDocumentation.tsx

```diff
@@ -99,7 +101,7 @@ export function SdkDocumentation({
 
   useEffect(() => {
     async function getGettingStartedDoc() {
-      const mod = isReplayOnboarding
+      const mod = isOldReplayOnboarding
         ? await import(
             /* webpackExclude: /.spec/ */
             `sentry/gettingStartedDocs/replay-onboarding/${platformPath}`
```

Comment:
Was this something we recently addded? (and customers are not seeing yet) because then we could just delete it and aim for only the new thing

Reply:
Oh you're right, I forgot this was feature flagged. Can remove this, but then I'd like to keep the check of `migrated = []` to keep track of which files are "good"

Reply:
ok, but i think at the end of this we're going a) only use migrated docs b) have all the js platforms migrated, so they work
c) i'm slightly worried that system will mean that all the BE docs need to be migrated too, but that'll just require sitting down and getting it done, so nbd.

Reply:
at the end, we won't even need the `migrated = []` because they'll all be migrated, this is just an intermediary step (even though it's feature flagged, so removing the array would only break for me...but peace of mind)

Reply:
this project should be renamed "overhaul the entire docs structure and a maybe perhaps update replay onboarding on the side"